apps/worker/src/app/workflow/usecases/send-message/send-message-in-app.usecase.ts

```diff
@@ -247,7 +247,7 @@ export class SendMessageInApp extends SendMessageBase {
     );
 
     await this.webSocketsQueueService.bullMqService.add(
-      `sendMessage-received-${message._id}`,
+      message._id,
```

Comment:
We will use the `messageId` as the `jobId` now we will have an unique event sent, therefore one unique job created.